templates/test/cpp/naming/resolver_component_tests_defs.include

```diff
@@ -43,6 +44,11 @@ args = argp.parse_args()
 def test_runner_log(msg):
   sys.stderr.write('\n%s: %s\n' % (__file__, msg))
 
+def python_args(arg_list):
+  if platform.system() == 'Windows':
+    return ['C:\\Python27\\python.exe'] + arg_list
```

Comment:
Please use [`sys.executable`](https://docs.python.org/2/library/sys.html#sys.executable) instead of hard-coding `C:\\Python27\\python.exe`.

Reply:
@rongjiecomputer done, thanks for pointing out